src/ripple/app/tx/impl/Remit.cpp

```diff
@@ -0,0 +1,545 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
```

Comment:
these copyrights are wrong and need to be updated in a number of places, but this PR isn't that place

Reply:
https://github.com/Xahau/xahaud/issues/282